templates/layout.php

```diff
@@ -0,0 +1,115 @@
+
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <title>
+        <?= $title ?>
+    </title>
+    <link href="./css/normalize.min.css" rel="stylesheet">
+    <link href="./css/style.css" rel="stylesheet">
+</head>
+<body>
+<div class="page-wrapper">
+
+<header class="main-header">
+    <div class="main-header__container container">
+        <h1 class="visually-hidden">YetiCave</h1>
+        <a class="main-header__logo">
+            <img src="./img/logo.svg" width="160" height="39" alt="Логотип компании YetiCave">
+        </a>
+        <form class="main-header__search" method="get" action="https://echo.htmlacademy.ru" autocomplete="off">
+            <input type="search" name="search" placeholder="Поиск лота">
+            <input class="main-header__search-btn" type="submit" name="find" value="Найти">
+        </form>
+        <a class="main-header__add-lot button" href="pages/add-lot.html">Добавить лот</a>
+
+        <nav class="user-menu">
+
+        <!-- show user info if $isAuth is true -->
+        <?php if($isAuth === 1) : ?>
```

Comment:
некритично, но лучше настроить Code Style PHPSTORM под [PSR-12](https://www.php-fig.org/psr/psr-12/).
Обычно после if while и т.д стоит пробел

```suggestion
        <?php if ($isAuth === 1) : ?>
```